src/chainparams.cpp

```diff
@@ -90,11 +86,17 @@ class CMainParams : public CChainParams {
         vSeeds.push_back(CDNSSeedData("xf2.org", "bitseed.xf2.org")); // Jeff Garzik
         vSeeds.push_back(CDNSSeedData("bitcoin.jonasschnelli.ch", "seed.bitcoin.jonasschnelli.ch")); // Jonas Schnelli
 
-        base58Prefixes[PUBKEY_ADDRESS] = std::vector<unsigned char>(1,0);
-        base58Prefixes[SCRIPT_ADDRESS] = std::vector<unsigned char>(1,5);
-        base58Prefixes[SECRET_KEY] =     std::vector<unsigned char>(1,128);
-        base58Prefixes[EXT_PUBLIC_KEY] = boost::assign::list_of(0x04)(0x88)(0xB2)(0x1E).convert_to_container<std::vector<unsigned char> >();
-        base58Prefixes[EXT_SECRET_KEY] = boost::assign::list_of(0x04)(0x88)(0xAD)(0xE4).convert_to_container<std::vector<unsigned char> >();
+        const unsigned char chPubkeyAddress[] = { 0 };
+        const unsigned char chScriptAddress[] = { 5 };
+        const unsigned char chSecretKey[] =     { 128 };
+        const unsigned char chExtPublicKey[] =  { 0x04, 0x88, 0xB2, 0x1E };
+        const unsigned char chExtSecretKey[] =  { 0x04, 0x88, 0xAD, 0xE4 };
```

Comment:
Here,
- only two years ago this was all changed to list_of in the name of newness: 8388289eb6dfc9add233f8f8b536f84caf771fa6
- Then you changed it to get rid of the global namespace 856e862f4a736fbdc38daae3b7f0fa34e1da317c
- Then some list_ofs had to be changed to std::vector again a2b04ddfe6452e7d6274f4096bf3f2aee695a6d9 in the name of c++11
... and now we're full circle, back to C-style initialization. Sorry for this, this may be just a personal annoyance against flip-flopping perfectly fine code, but I feel this is not helping anyone :)